src/app/school/school-page.css

```
@@ -1 +1,844 @@
+}
+
+.school-page-attendance-hero small {
+  display: block;
+  margin-top: 4px;
+  color: var(--sn-muted);
+  font-size: 12px;
+}
+
+.school-page-attendance-ring {
+  --progress: 70%;
+  width: 130px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)